### PR TITLE
DPC-787: Release postman collection to DPC users

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-787](https://jira.cms.gov/browse/DPC-787)

Work with the H team to make the postman collection available to external DPC users (language updates on the website)

### Proposed Changes
- Add download link to postman collection
- Add postman collection section to documenation

### Change Details
- Updated about pilot section of home page

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2020-10-27 at 10 40 22 AM

### CODE REVIEW

File: dpc-web/config/routes.rb [hunk 1]
```diff
@@ -53,6 +53,7 @@
   match '/download_jwt_tool', to: 'application#download_jwt_tool', as: 'download_jwt_tool', via: :post
   match '/download_prac_json', to: 'application#download_prac_json', as: 'download_prac_json', via: :post
   match '/download_pt_json', to: 'application#download_pt_json', as: 'download_pt_json', via: :post
+  match '/download_postman_collection', to: 'application#download_postman', as: 'download_postman', via: :post
```
Comment (whuang85) [inline dpc-web/config/routes.rb:56]: curious why downloads are posts

**Review by whuang85 — APPROVED** [review]

File: dpc-web/app/views/pages/reference.md [hunk 2]
```diff
@@ -1817,3 +1817,50 @@ To obtain the exported explanation of benefit data, a GET request is made to the
...
+- A registered client token
+- Your private key
+- Your public key ID
+
+1. Please download the DPC Postman collection found in the left sidebar. This will include the collection of requests, the sandbox environment, and global variables to be imported into your Postman App.
+2. Select the environment (top right): Data at the Point of Care Sandbox
+3. Please fill in the following values:
+    - client_token: Your [client token](#step-two-client-tokens) is generated through the DPC portal. Be sure to save a copy of your token in a safe place.
+    - PRIVATE_KEY: Paste the contents of your private key in the `PRIVATE_KEY` field of your local Postman sandbox environment. Do not share your private key otherwise. If you do not already have your public and private keys, please [generate your public/private key pair](#step-three-public-keys) through the DPC portal.
+    - key-id: This is the system id of your public key, which is returned to you when the public key is uploaded to the DPC portal. You need this to generate a JWT, which will be exchanged for an access token.
```
Comment (jonfulk) [inline dpc-web/app/views/pages/reference.md:1836]: Not part of this PR, but I'm regretting that the variable is `key-id` instead of `key_id`

File: dpc-web/app/views/pages/reference.md [hunk 3]
```diff
@@ -1817,3 +1817,50 @@ To obtain the exported explanation of benefit data, a GET request is made to the
...
+
+With these 3 values in place, the JWT and a fresh access token are automatically generated for you before each request in this Postman collection to prevent you from having to manually refresh the access token every 5 minutes while using the collection. You may occasionally  receive a 401 error message regarding invalid credentials. If this happens, please try your request a second time to fix the error.
```
Comment (jonfulk) [inline dpc-web/app/views/pages/reference.md:1838]: I would remove `to fix the error` from this sentence and just end it at `a second time`. If that doesn't work, there may be another problem, so it may not "fix" the error.

**Review by jonfulk — DISMISSED** [review]
A couple minor comments, but looks good! I'm eager to get this out and to hear whether our users find it helpful.

File: dpc-web/app/views/pages/reference.md (hunk 3 above)
Comment (Sun-Mountain) [reply dpc-web/app/views/pages/reference.md:1838]: Will run by Crystal

Comment (Sun-Mountain) [reply dpc-web/app/views/pages/reference.md:1838]: H Team has final say on the language

**Review by jonfulk — APPROVED** [review]
Looks good!

File: dpc-web/app/views/pages/reference.md [hunk 4]
```diff
@@ -1817,3 +1817,50 @@ To obtain the exported explanation of benefit data, a GET request is made to the
...
+2. Select the environment (top right): Data at the Point of Care Sandbox
+3. Please fill in the following values:
+    - client_token: Your [client token](#step-two-client-tokens) is generated through the DPC portal. Be sure to save a copy of your token in a safe place.
+    - PRIVATE_KEY: Paste the contents of your private key in the `PRIVATE_KEY` field of your local Postman sandbox environment. Do not share your private key otherwise. If you do not already have your public and private keys, please [generate your public/private key pair](#step-three-public-keys) through the DPC portal.
+    - key-id: This is the system id of your public key, which is returned to you when the public key is uploaded to the DPC portal. You need this to generate a JWT, which will be exchanged for an access token.
+
+With these 3 values in place, the JWT and a fresh access token are automatically generated for you before each request in this Postman collection to prevent you from having to manually refresh the access token every 5 minutes while using the collection. You may occasionally  receive a 401 error message regarding invalid credentials. If this happens, please try your request a second time.
```
Comment (SalGutierrez) [inline dpc-web/app/views/pages/reference.md:1838]: Not really in an issue. : "You may occasionally  <There is an extra space here>receive a 401 error"

**Review by SalGutierrez — APPROVED** [review]

**Review by SalGutierrez — APPROVED** [review]
